### DISCUSSION TITLE
萨米肉鸽脚本点开行商那里就不继续操作了

### DISCUSSION THREAD

**Discussion by shinetse945** [MaaAssistantArknights/MaaAssistantArknights#7439] [Q&A]
今天刚出现这个问题  而且出现2次了   之前从来没碰到过    是怎么回事呢     我只开了自动唤醒和自动肉鸽   肉鸽用的特训分队  策略是刷等级

**Comment by zzyyyl** [comment]
去 [issue](https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose) 开 [bug](https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new?assignees=&labels=bug&projects=&template=cn-bug-report.yaml) 帖发日志
